applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.h

```diff
@@ -243,16 +243,23 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) ConstitutiveLawUtilities
         }
     }
 
-        /**
+    /**
     * It calculates the constitutive matrix C in plane stress, strain or in 3D
     * @param C: The constitutive matrix
     * @param rValues Parameters of the constitutive law
     */
-    static void CalculateElasticMatrixPlaneStress(MatrixType &C, ConstitutiveLaw::Parameters &rValues);
-    static void CalculateElasticMatrixPlaneStrain(MatrixType &C, ConstitutiveLaw::Parameters &rValues);
-    static void CalculateElasticMatrix(MatrixType &C, ConstitutiveLaw::Parameters &rValues);
+    static void CalculateElasticMatrixPlaneStress(MatrixType &C, const double E, const double NU);
+    static void CalculateElasticMatrixPlaneStrain(MatrixType &C, const double E, const double N);
+    static void CalculateElasticMatrix(MatrixType &C, const double E, const double N);
```

Comment:
```suggestion
    static void CalculateElasticMatrixPlaneStrain(MatrixType &C, const double E, const double NU);
    static void CalculateElasticMatrix(MatrixType &C, const double E, const double NU);
```

Or even better, use the full names (my recommendation)

Reply:
done